bitnami/postgresql-ha/templates/postgresql/witness-statefulset.yaml

```diff
@@ -55,8 +48,8 @@ spec:
       affinity: {{- include "common.tplvalues.render" (dict "value" .Values.witness.affinity "context" $) | nindent 8 }}
       {{- else }}
       affinity:
-        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.witness.podAffinityPreset "component" "postgresql" "context" $) | nindent 10 }}
-        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.witness.podAntiAffinityPreset "component" "" "context" $) | nindent 10 }}
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.witness.podAffinityPreset "component" "postgresql-witness" "customLabels" $podLabels "context" $) | nindent 10 }}
```

Comment:
Why are you changing `postgresql` to `postgresql-witness`?

Reply:
fixed at https://github.com/bitnami/charts/pull/18410/commits/0a09ba9f9564b2336a69c375ce06d66987b0260a

Reply:
Thanks!